projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.ts

```diff
@@ -17,17 +17,18 @@
 
 import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
 import { RecordUiService } from '@rero/ng-core';
+import { ItemHoldingsCallNumberPipe, UserService } from '@rero/shared';
 import { BsModalService } from 'ngx-bootstrap/modal';
 import { ItemsService } from 'projects/admin/src/app/service/items.service';
 import { RecordPermissionService } from 'projects/admin/src/app/service/record-permission.service';
 import { forkJoin } from 'rxjs';
 import { first } from 'rxjs/operators';
-import { UserService } from '@rero/shared';
 import { ItemRequestComponent } from '../../item-request/item-request.component';
 
 @Component({
   selector: 'admin-default-holding-item',
-  templateUrl: './default-holding-item.component.html'
+  templateUrl: './default-holding-item.component.html',
+  providers: [ItemHoldingsCallNumberPipe]
```

Comment:
Seems not yet be use in the component, should be removed (and remove import).
But it depends of the next comment (overkill component)